src/system-health/health_checker/hardware_checker.py

```diff
@@ -82,6 +82,7 @@ def _check_fan_status(self, config):
             self.set_object_not_ok('Fan', 'Fan', 'Failed to get fan information')
             return
 
```

Comment:
@Junchao-Mellanox please update the description of the _check_fan_status() to include fan direction check

Reply:
fixed